python/pyarrow/src/arrow/python/inference.cc

```diff
@@ -77,6 +79,8 @@ Status ImportPresentIntervalTypes(OwnedRefNoGIL* interval_types_tuple) {
 
 }  // namespace
 
+int import_pyarrow();
```

Comment:
```suggestion
import_pyarrow();
```

Reply:
This change produced an error:
```
/arrow/python/pyarrow/src/arrow/python/inference.cc
  /arrow/python/pyarrow/src/arrow/python/inference.cc:82:17: error: expected constructor, destructor, or type conversion before ';' token
     82 | import_pyarrow();
        |   
```

I do not fully understand the usage of this function. But if we need to call it before using any other function in the pyarrow C++ we could do an if statement or something?

Reply:
Removing `import_pyarrow();` altogether also works so I went with that.